src/framework/theme/component/modal/modal.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { render } from 'react-native-testing-library';
+import {
+  View,
+  Text,
+} from 'react-native';
+import { ModalPanel } from './modalPanel.component';
+import { ModalComponent } from './modal.component';
+
+describe('@ modal component/panel checks', () => {
+
+  class ModalPanelTest extends React.PureComponent {
```

Comment:
Simplify this to Stateless Function Component like `Mock` syntax we used in other tests

Reply:
No needs I guess. This will cause difficulties in checking life-cycle hooks calling